lib/alchemy/test_support/factories/page_factory.rb

```diff
@@ -4,27 +4,14 @@
 FactoryGirl.define do
 
   factory :alchemy_page, class: 'Alchemy::Page' do
-    language { Alchemy::Language.default || FactoryGirl.create(:alchemy_language) }
+    language { Alchemy::Language.default }
```

Comment:
Style/BlockDelimiters: Prefer `do...end` over `{...}` for procedural blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)